src/main/resources/messages/KelpBundle.properties

```diff
@@ -0,0 +1,22 @@
+kelpIntentionsFamilyName=Kelp intentions
+kelpNotificationGroup=Kelp
+openingInDemoAppMessage=Opening in demo app
+turningOnScreenMessage=Turning the screen on
+chooseDevicePopupTitle=Choose a Device
+checkingInstalledAppVersionMessage=Checking installed demo app version
+uninstallingPreviousAppMessage=Uninstalling previous demo app version
+installingAppMessage=Installing demo app apk
+demoAppLatestVersionInfoFileNotFoundErrorMessage=File that contains info about latest version of the demo app wasn't found: {0}
+demoAppLatestVersionNotFoundErrorMessage=Latest version of the demo app wasn't found by ths regex '{0}' in this file {1}
+apkFileNotFoundErrorMessage=Demo apk file was not found here: {0}
+openedNotificationContent={0} opened in demo app
+componentOpeningErrorTitle=Error While Opening DS Component
+noSuchComponentNotificationContent=Couldn't determine the fully qualified name of the design system component
+noRunningDevicesNotificationTitle=No running devices found
+noRunningDevicesNotificationContent=Please, start one, or wait for it to finish loading
+noAndroidSdkNotificationContent=No android sdk found. Please, install it
+
+openInDemoAppIntentionName={0} Open in design system demo app
```

Comment:
```suggestion
openInDemoAppIntentionName=\uD83D\uDE80 Open in design system demo app
```
This works. Used this action to paste as unicode:

<img width="536" alt="Screenshot 2024-02-02 at 12 03 10" src="https://github.com/ozontech/kelp/assets/15000556/7e0044d1-2b56-404c-83b6-2d4c273691bd">